src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,37 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import javax.swing.text.Element;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By userAvatarIcon = By.cssSelector("img.avatar");
+
+    By selectedPlayList = By.xpath("//li[@class= 'playlist playlist']//a[contains(text(), 'My Play List')]");
+    By selectedPLField = By.cssSelector("[name = 'name']");
+    By renamedPlayList = By.xpath("//div[@class= 'success show']");
+
+    @FindBy(xpath = "//li[@class= 'playlist playlist']//a[contains(text(), 'My Play List')]")
+    WebElement selectedListLocator;
+    @FindBy(css = "[name = 'name']")
+    WebElement selectedNewNameLocator;
+    @FindBy(xpath = "//div[@class= 'success show']")
+    WebElement renamedPLocator;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```